sveltekit/src/routes/+page.svelte

```diff
@@ -0,0 +1,16 @@
+<script>
+  import TvChartContainer from "../components/TVChartContainer.svelte";
+  import { version } from "$lib/charting_library";
+</script>
+
+<p>Library version: {version()}</p>
+
+<TvChartContainer />
+
+<style>
+  p {
+    color: rgb(124, 89, 7);
+    font-family: "Comic Sans MS", cursive;
```

Comment:
Could we change this to:
```suggestion
    font-family: -apple-system, BlinkMacSystemFont, 'Trebuchet MS', Roboto, Ubuntu, sans-serif;
```